recipes/e3sm-unified/build.bash

```diff
@@ -11,6 +11,6 @@ if [ $upload == "True" ]
 then
    for python in "${pythons[@]}"
    do
-      anaconda upload -u e3sm /home/xylar/miniconda3/conda-bld/linux-64/e3sm-unified-${version}-py${python}_${build}.tar.bz2
+      anaconda upload -u e3sm /home/${USER}/miniconda3/conda-bld/linux-64/e3sm-unified-${version}-py${python}_${build}.tar.bz2
```

Comment:
```suggestion
      anaconda upload -u e3sm ${HOME}/miniconda3/conda-bld/linux-64/e3sm-unified-${version}-py${python}_${build}.tar.bz2
```

Reply:
Maybe uncomment the conda build command while you're at it.

Reply:
Maybe set `upload=False` here as well

Reply:
Done! See dcd9ac9c966474cc3132dccabbc8032fecd2ec87